bottlecap/src/event_bus/constants.rs

```diff
@@ -0,0 +1 @@
+pub const MAX_EVENTS: usize = 100;
```

Comment:
Why 100?

Reply:
Just a number I picked. We can make it configurable, I can add a TODO item and ticket

Reply:
Yeah – I'm just afraid we might drop data in scenarios where customer is sending tons of metrics + logs + eventually traces

Reply:
doesn't drop, just blocks until the messages are acknowledged by the main thread.

Reply:
Ah thank you, that's good to know! Thank you for explaining it